client/src/main.c

```diff
@@ -1126,6 +1131,7 @@ static void handleWindowLeave()
 static void handleWindowEnter()
 {
   g_cursor.inWindow = true;
+  g_state.focused   = true;
```

Comment:
Entered is not the same as focused, this is incorrect. Enter means the mouse is over the window, but the window may not have input focus.

Reply:
Fixed.